src/vue/backend/src/templates/omnibarForm.vue

```diff
@@ -1,158 +1,143 @@
 <template>
```

Comment:
Page is not matching designs in terms of lack of bold titles and toggle text styles. Also the border line is too dark, should be #F2F3F4

Reply:
I tried font-weight: 600 and it looked too bold, so I dialed it back to 500 which seemed closer to the design. I might need a ruling from the designer on which one to choose.